README.md

```diff
@@ -36,11 +36,11 @@ and [contributing guidelines](CONTRIBUTING.md).
 
 2. FerretDB does not support NUL (`\0`) characters in strings.
 
-3. Collection names restrictions:
+3. Database and collection names restrictions:
 
 * collection name cannot start with the reserved prefix `_ferretdb_`.
 * collection name must not include non-latin letters, spaces, dots, dollars or dashes.
-* collection name length must be less or equal than 120 symbols.
+* collection name length must be less or equal than 120 symbols, database name length limit is 64 symbols.
```

Comment:
So, there is only one restriction for databases? It's not mentioned that database should match regex and shouldn't start from a reserved prefix. Based on this doc, only collection name has such restriction.

Reply:
> P.S. @seeforschauer could you also check `MsgUpdate`? I think it calls collection creation somewhere under the hood as well.

WDYM? We here about the database validation. 
Anyway, `MsgUpdate` uses `insert` private handler function if necessary so it definitely has these checks.

Reply:
Don't [this place](https://github.com/seeforschauer/FerretDB/blob/issue-807-validate-db-names/internal/handlers/pg/msg_update.go#L65) need a similar error check as the one you added for `MsgCreate` and `MsgInsert`?

Reply:
oh thank you, I understood why I didn't find this:
there is a deprecated usage.

![Screenshot from 2022-07-19 13-08-58](https://user-images.githubusercontent.com/4604088/179725981-a7c95a45-0ece-41fa-a330-e32e3ed737e1.png)